guerrilla_unix.go

```diff
@@ -11,5 +11,5 @@ func getFileLimit() (uint64, error) {
 	if err != nil {
 		return 0, err
 	}
-	return uint64(rLimit.Max), nil
+	return rLimit.Max, nil
```

Comment:
We actually need to cast it to uint64
Under Linux and most other unix-like OS, the type is already uint64, so it may appear this cast is unnecessary.
However, FreeBSD, the type is different. Learned this oddity here: https://github.com/flashmob/go-guerrilla/commit/421a18f0dda6c8117e8914a993cfc9c489d715df

Perhaps a neater solution would be to create a different version of this code for freebsd (guerrilla_freebsd.go) with an exclusive `freebsd` build tag.

Reply:
True, sorry I did not consider that was the reason so I will add a comment that its indeed needed for FreeBSD.